src/App.tsx

```diff
@@ -15,71 +16,102 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type Props = {};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+interface State {
+  selectedGood: string | null,
+  isSelected: boolean,
+}
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export class App extends React.Component<Props, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+    isSelected: true,
```

Comment:
`isSelected` is redundant. You can just check if selectedGood is null, you don't need additional variable to track this state